lib/play.js

```diff
@@ -0,0 +1,42 @@
+const { getTeamChoice } = require("./chooseTeam");
+const {
+  setTeamChoice,
+  setUserGridChoice,
+  setCpuGridChoice,
+} = require("./store");
+const { pickOrder, playOrder } = require("./playOrder");
+const logGrid = require("./logGrid");
+const { getUserGridChoice } = require("./chooseUserGrid");
+const { getCpuGridChoice } = require("./chooseCpuGrid");
+const { checkForGameOver } = require("./checkWin");
+
+const userTurn = async (store) => {
+  setUserGridChoice(store, await getUserGridChoice(store));
```

Comment:
[optional style thing] it's a bit easier to read top-to-bottom if you use temporary variables, and also easier to insert breakpoints if you need to debug.

At el banco, there seemed to be a rule of not doing `thing(await otherThing())`, and always saving the result of an await to a variable (was fine to do `thing(thirdThing())` though, I think).
```suggestion
  const choice = await getUserGridChoice(store);
  setUserGridChoice(store, choice);
```

Style choices are up to you as the maintainer though, so think of this as just an observation.